hackathon_site/dashboard/frontend/src/slices/order/orderSlice.test.ts

```diff
@@ -0,0 +1,38 @@
+import store, { makeStore, RootState } from "slices/store";
+import {
+    orderReducerName,
+    orderSelectors,
+    orderSliceSelector,
+    isLoadingSelector,
+    initialState,
+} from "./orderSlice";
+
+const mockState: RootState = {
+    ...store.getState(),
+    [orderReducerName]: initialState,
+};
+
+describe("Selectors", () => {
+    test("teamSliceSelector returns the team store", () => {
```

Comment:
this is not the team slice, change the test name to "orderSliceSelector returns the order store"